dolphin/api/v1/volumes.py

```diff
@@ -11,20 +11,50 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import six
+from webob import exc
 
+from dolphin import db, exception
+from dolphin.api import api_utils
 from dolphin.api.common import wsgi
+from dolphin.api.views import volumes as volume_view
 
 
 class VolumeController(wsgi.Controller):
 
+    def __init__(self):
+        super(VolumeController, self).__init__()
+        self.search_options = ['name', 'status', 'id', 'storage_id', 'pool_id']
```

Comment:
Volume model does not have 'pool_id', please check.

Reply:
correct, Have added 'original_pool_id'. added 2 more fields  'wwn' and 'original_id' also . I hope these make sense.
Need to finalize and update API spec. created an issue for API spec update.
https://github.com/sodafoundation/SIM/issues/48